app/components/provider_interface/diversity_information_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<% if display_diversity_information? -%>
+  <%= render SummaryListComponent.new(rows: rows) %>
+<% else -%>
+  <p class="govuk-body"><%= message %></p>
+  <% unless message == 'No information shared' -%>
```

Comment:
Might it be better to switch on `diversity_information_declared?` here rather than the string?